scripts/variantstore/wdl/extract/import_gvs.py

```diff
@@ -312,18 +312,30 @@ def convert_array_with_id_keys_to_dense_array(arr, ids, drop=[]):
     info(f'import_gvs: using target_records (records per partition) of {target_records} for VDS merge')
 
     interval_tmp = os.path.join(tmp_dir, 'interval_checkpoint.ht')
+    if hl.hadoop_exists(interval_tmp):
+        info(f'import_gvs: interval checkpoint table "{interval_tmp}" already exists, deleting')
+        hl.current_backend().fs.rmtree(interval_tmp)
+
     target_final_intervals, _ = calculate_new_intervals(first_ref_mt, target_records, interval_tmp)
 
     with hl._with_flags(no_whole_stage_codegen='1'):
 
         merge_tmp = os.path.join(tmp_dir, 'merge_tmp.vds')
-        info(f'import_gvs: calling Hail VDS combiner for merging {len(vds_paths)} intermediates')
-        combiner = hl.vds.new_combiner(output_path=merge_tmp,
-                                       vds_paths=vds_paths,
-                                       target_records=target_records,
-                                       temp_path=tmp_dir,
-                                       use_genome_default_intervals=True)
-        combiner.run()
```

Comment:
this being deleted worries me a bit--this feels like a hard coding one off that was taken from Miguel and I trying to push a semi-successful run through. Let's discuss